### ISSUE TITLE
[EN][Anduin Wrynn] 

### ISSUE THREAD

**Issue by Aruzam** [Atlantiss/BugTracker#1899]
http://www.wowhead.com/npc=44293/anduin-wrynn
This NPC starts following you soon after you take the Hero's call for Twilight Highlands and talk to the king in Stormwind. Problem is, he's supposed to fly around with you completing the few quests in Stormwind quest-line leading to Twilight Highlands, while atm he's only walking through the town following you. This makes him much slower to drag around and poses a few problems completing the quests given in Stormwind. 
To sum it up, NPC is supposed to be flying and follow you wherever you go, currently he's only walking on foot.

**Comment by Wolffenstein** [comment]
Weird, he had his mounts scripted and it was working fine.

**Comment by Aruzam** [comment]
As an additional note, I failed to mention I'm a druid. In case that changes anything (maybe because I don't use an actual mount to fly, instead I just shapeshift?)

**Comment by Navedonar** [comment]
http://atlantiss.eu/changelog#rev4347

**Comment by Wolffenstein** [comment]
Fixed in rev 4347.
